src/java/com/github/ruleant/getback_gps/AbstractGetBackGpsActivity.java

```diff
@@ -280,6 +280,75 @@ public void onClick(final DialogInterface dialog,
                                 // User cancelled the dialog
                             }
                         });
+        // Create the AlertDialog object and display it
+        builder.create().show();
+    }
+    public final void enterLocation() {
+        if (mBound && mService.getLocation() == null) {
```

Comment:
the mService.getLocation() is not necessary for this dialog, because all data is entered manually.
This in contrast to the storeLocation dialog, were the current Location is stored, so the check is necessary there.